go-controller/pkg/ovn/egressip.go

```diff
@@ -2451,11 +2451,18 @@ func (e *EgressIPController) deletePodEgressIPAssignment(ni util.NetInfo, egress
 		return err
 	}
 	var ops []ovsdb.Operation
-	if !loadedPodNode || isLocalZonePod { // node is deleted (we can't determine zone so we always try and nuke OR pod is local to zone)
+	// For CDN only, add SNATs to support external GW feature
+	if ni.IsDefault() && (!loadedPodNode || isLocalZonePod) {
```

Comment:
this is the per pod SNATs huh? ok makes sense since we don't have it for UDNs..but its a bit weird small catches like this are slipping into this PR :D looks like we have some gaps in testing that can come later. I consider this fix outside this PR's scope but OK to get this in logic wise